cli/cmd/team_members.go

```diff
@@ -0,0 +1,403 @@
+			  Team members can also be granted organization-level roles.
+`,
+	}
+
+	// list command is used to list all lacework team members
+	teamMembersListCommand = &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "List all team members",
+		Long:    "List all team members configured in your Lacework account.",
+		Args:    cobra.NoArgs,
+		RunE: func(_ *cobra.Command, _ []string) error {
+			cli.StartProgress(" Fetching team members...")
+			teamMembers, err := cli.LwApi.V2.TeamMembers.List()
+			cli.StopProgress()
+			if err != nil {
+				return errors.Wrap(err, "unable to get team members")
+			}
+			if len(teamMembers.Data) == 0 {
+				msg := `There are no team members configured in your account.
+
+Get started by configuring your team members using the command:
+
+    lacework team-member create
+
+If you prefer to configure team members via the WebUI, log in to your account at:
+
+    https://%s.lacework.net
+
+Then navigate to Settings > Team Members.
+`
+				cli.OutputHuman(fmt.Sprintf(msg, cli.Account))
+				return nil
+			}
+			if cli.JSONOutput() {
+				return cli.OutputJSON(teamMembers)
+			}
+
+			var rows [][]string
+			for _, tm := range teamMembers.Data {
+				rows = append(rows, []string{tm.UserGuid, tm.UserName, enabled(tm.UserEnabled)})
+			}
+
+			cli.OutputHuman(renderSimpleTable([]string{"GUID", "NAME", "ENABLED"}, rows))
+			return nil
+		},
+	}
+	// show command is used to retrieve a lacework team member by guid
+	teamMembersShowCommand = &cobra.Command{
+		Use:   "show",
+		Short: "Show a team member by id",
+		Long:  "Show a single team member by it's id.",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			var response api.TeamMemberResponse
+			cli.StartProgress(" Fetching team member...")
+
+			err := cli.LwApi.V2.TeamMembers.Get(args[0], &response)
+			if err != nil {
+				cli.StopProgress()
+				return errors.Wrap(err, "unable to get team member")
+			}
+			cli.StopProgress()
+
+			if cli.JSONOutput() {
+				return cli.OutputJSON(response)
+			}
+
+			teamMember := response.Data
+			headers := [][]string{{teamMember.UserGuid, teamMember.UserName, enabled(teamMember.UserEnabled)}}
+			cli.OutputHuman(renderSimpleTable([]string{"GUID", "NAME", "STATUS"}, headers))
+			cli.OutputHuman("\n")
+			cli.OutputHuman(buildTeamMemberDetailsTable(teamMember))
+
+			return nil
+		},
+	}
+
+	// delete command is used to remove a lacework team member by id
+	teamMembersDeleteCommand = &cobra.Command{
+		Use:   "delete",
+		Short: "Delete a team member",
+		Long:  "Delete a single team member by it's ID.",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			cli.StartProgress(" Deleting team member...")
+			err := cli.LwApi.V2.TeamMembers.Delete(args[0])
+			cli.StopProgress()
+			if err != nil {
+				return errors.Wrap(err, "unable to delete team member")
+			}
+			cli.OutputHuman("The team member with GUID %s was deleted\n", args[0])
```

Comment:
Could we agree on a pattern for delete operations? We can clearly see that many engineers print this message in different forms:
```
cli/cmd/alert_rules.go:                 cli.OutputHuman(fmt.Sprintf("The alert rule with GUID %s was deleted \n", args[0]))
cli/cmd/integration.go:                 cli.OutputHuman("The integration %s was deleted.\n", args[0])
cli/cmd/lql_delete.go:  cli.OutputHuman("The query %s was deleted.\n", args[0])
cli/cmd/policy.go:              fmt.Sprintf("The policy %s was deleted.\n", args[0]))
cli/cmd/report_rules.go:                        cli.OutputHuman("The report rule with GUID %s was deleted\n", args[0])
cli/cmd/team_members.go:                        cli.OutputHuman("The team member with GUID %s was deleted\n", args[0])
```